getting-started/windows-calico/rancher.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Install Calico for Windows on a Rancher Kubernetes Engine cluster
+description: Install Calico for Windows on a Rancher RKE cluster.
+canonical_url: '/getting-started/windows-calico/rancher'
+---
+
+### Big picture
+
+Install {{site.prodnameWindows}} on a Rancher Kubernetes Engine (RKE) cluster.
+
+### Value
+
+Run Linux and Windows workloads on a RKE cluster with {{site.prodname}}.
+
+### Before you begin
+
+**You will need**
+
+- An RKE cluster provisioned with {% include open-new-window.html text='no network plugin' url='https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins#disabling-deployment-of-a-network-plug-in' %}
+but which otherwise meets the {{site.prodnameWindows}} Kubernetes [cluster requirements]({{site.baseurl}}/getting-started/windows-calico/requirements). This guide was tested with RKE v1.18.9.
+- One or more Windows nodes that meet the [requirements]({{site.baseurl}}/getting-started/windows-calico/requirements).
+
+### How to
+
+The following steps will outline the installation of {{site.prodname}} networking on the RKE cluster, then the installation of {{site.prodnameWindows}} on the Windows nodes.
+
+1. Install the Tigera {{site.prodname}} operator and custom resource definitions.
+
+   ```
+   kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
+   ```
+
+1. Download the necessary Installation custom resource.
+
+   ```bash
+   wget {{ "/manifests/custom-resources.yaml" | absolute_url }}
+   ```
+
+1. Update the `calicoNetwork` options, ensuring that the correct pod cidr is set. (Rancher uses `10.42.0.0/16` by default.)
+   Below are example installations for VXLAN and BGP networking using the default Rancher pod cidr:
```

Comment:
Below are sample installations for VXLAN and BGP networking using the default Rancher pod CIDR: